src/io/flutter/FlutterBundle.properties

```diff
@@ -117,7 +118,8 @@ flutter.module.create.settings.help.org.label=Organization:
 flutter.module.create.settings.help.org.description=Enter the organization domain name in reverse domain notation.
 flutter.module.create.settings.help.project_type.label=Project type:
 flutter.module.create.settings.help.project_type.description.app=Select an "Application" when building for end users.
-flutter.module.create.settings.help.project_type.description.plugin=Select a "Plugin" when building system hooks for developers.
+flutter.module.create.settings.help.project_type.description.plugin=Select a "Plugin" when exposing an Android or iOS API for developers.
```

Comment:
I would really like to keep these "help" strings consistent with [`flutter --help create`](https://github.com/flutter/flutter/blob/41378e20322b9fc4c317bfd1b4c33beeafcd6981/packages/flutter_tools/lib/src/commands/create.dart#L45) and the [website docs](https://flutter.io/developing-packages/#types).

Also, `when creating a cross-platform component` is confusing; a plugin package can also be cross-platform (i.e. support both iOS and Android).

But would love suggestions for how to improve the flutter/website ones also!